src/components/sideNavigation/tests/SideNavigationLoadingItem.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import { shallow, ReactWrapper, mount } from 'enzyme';
+import { SideNavigationLoadingItem, ISideNavLoadingItemProps } from '../SideNavigationLoadingItem';
+
+describe('<SideNavigationLoadingItem />', () => {
+  let wrapper: ReactWrapper<ISideNavLoadingItemProps, any>;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <SideNavigationLoadingItem />,
+      { attachTo: document.getElementById('App') }
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+    wrapper.detach();
+  });
+
+  it('should render a <SideNavigationLoadingItem /> with mod-sub-navigation-left-margin', () => {
+    let container = wrapper.find('div').first();
+    expect(container.hasClass('mod-sub-navigation-left-margin')).toBe(true);
+  });
+
+  it('should render a <SideNavigationLoadingItem /> with navigation-loading-item-grey', () => {
+    let container = wrapper.find('div').first();
+    expect(container.hasClass('navigation-loading-item-grey')).toBe(true);
+  });
+
+  it('should render a <SideNavigationLoadingItem /> with classes prop', () => {
+    const classes = 'mod-width-30';
+    let container = wrapper.find('div').first();
+    wrapper.setProps({ classes: classes });
```

Comment:
ES6 has a cool feature supported by TypeScript called [Object Literal shorthand](https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand). 
Basically, instead of: `{ name: name, classes: classes`, you can write `{ name, classes }`. It will mean the same thing.

There is also a linter rule that we can add for that (for another time).